src/io/opendss.jl

```diff
@@ -862,23 +146,22 @@ function dss2tppm_load!(tppm_data::Dict, dss_data::Dict, import_all::Bool)
 
     if haskey(dss_data, "load")
         for load in dss_data["load"]
-            defaults = get_prop_default("load")
-            merge!(defaults, load)
+            defaults = createLoad(load["bus1"], load["name"]; to_sym_keys(load)...)
 
             loadDict = Dict{String,Any}()
 
             nphases = tppm_data["phases"]
             name, nodes = parse_busname(defaults["bus1"])
 
-            loadDict["name"] = defaults["id"]
+            loadDict["name"] = defaults["name"]
             loadDict["load_bus"] = find_bus(name, tppm_data)
-            loadDict["pd"] = PMs.MultiPhaseVector(phase_on_off(fill(defaults["kw"], nphases), nodes, nphases))
-            loadDict["qd"] = PMs.MultiPhaseVector(phase_on_off(fill(defaults["kvar"], nphases), nodes, nphases))
+            loadDict["pd"] = PMs.MultiPhaseVector(parse_array(defaults["kw"] / 1e3, nodes, nphases))
+            loadDict["qd"] = PMs.MultiPhaseVector(parse_array(defaults["kvar"] / 1e3, nodes, nphases))
```

Comment:
Where does this `1e3` some from?  To convert `kw` into `mw`?

Reply:
yes

Reply:
I haven't tried that yet, but since the minimal set of components so far are already converted, I would propose keeping it as is, and I can make a new Issue about exploring this possibility.

Reply:
Sounds good to me.